### ISSUE TITLE
The AWS Access Key Id you provided does not exist in our records.

### ISSUE THREAD

**Issue by Albert2522** [t3chnoboy/amazon-product-api#85]
Hi
I checked my keys on a scratchpad but I still get this error whether it itemSearch or ItemLookUp

{ '$': { xmlns: 'http://ecs.amazonaws.com/doc/2013-08-01/' },
  Error: 
   [ { Code: [ 'InvalidClientTokenId' ],
       Message: [ 'The AWS Access Key Id you provided does not exist in our records.' ] } ],
  RequestId: [ 'ad5243d3-04b2-4bd7-8908-82789ed48233' ] }

Code: 

const amazon_client = amazon.createClient({
  awsID: 'smthg',
  awsSecret: 'smthg',
  awsTag: 'smthg'
});

amazon_client.itemLookup({
  idType: 'UPC',
  itemId: '884392579524'
    }).then(function(results) {
      console.log(JSON.stringify(results));
    }).catch(function(err) {
      console.log(util.inspect(err, false, null));
});

**Comment by nick-preda** [comment]
where do you actually try to get the keys from?

there are so many from where to choose from

**Comment by budda** [comment]
Make sure you have specified the domain endpoint to match where your access key is registered for.
e.g.

`domain: 'webservices.amazon.co.uk'`

**Comment by AnthoniG** [comment]
@budda Hi, just about to start using the library for a project I have an idea for.

I want to offer the ability to search multiple Amazon 'areas'
Does this mean I will have to register with Amazon for each of those countries ?
(ie like .ca, .za, etc) ?

**Comment by budda** [comment]
@AnthoniG yeah your credentials (key) needs to be registered for the country you want to search in, otherwise you'll get an API error.

**Comment by AnthoniG** [comment]
@budda Thanks for that.
Guess looks like got some more registration forms to fill out then.
One for each country I want to search in (which is a few lol)

Can I use the same account I used before?
I registered .co.uk (because that is where I am based), so can I use that same one to register for .ca, .it, .za, etc ?
